src/plugins/visualizations/public/wizard/group_selection/group_selection.tsx

```diff
@@ -224,15 +225,15 @@ const ToolsGroup = ({ visType, onVisTypeSelected, showExperimental }: VisCardPro
         <EuiIcon type={visType.icon || 'empty'} size="l" />
       </EuiFlexItem>
       <EuiFlexItem>
-        <EuiFlexGroup gutterSize="xs" alignItems="center" responsive={false}>
+        <EuiFlexGroup gutterSize="s" alignItems="center" responsive={false}>
```

Comment:
why this change?

Reply:
Thanks so much for the review despite the fact that you are off for the day, @flash1293! It's greatly appreciated ❤️ 

This change was so that the visualization name and the new "Deprecated" tag had a tiny bit more space between them, like so: 

![image](https://user-images.githubusercontent.com/8698078/170100216-8d977d59-6f4c-43eb-8612-896a7d6a487c.png)